src/popup/components/AssetIcon.vue

```diff
@@ -26,7 +26,7 @@ import EthereumIcon from '@/icons/coin/ethereum.svg?vue-component';
 import LexonTokenIcon from '@/icons/tokens/ct_xtk8rSz9suPb6D6VLquyfVji25FcnFRDjn3dnn5mmvHsPiESt.svg?vue-component';
 import ChainlinkTokenIcon from '@/icons/tokens/0x779877a7b0d9e8603169ddbd7836e478b4624789.svg?vue-component';
 
-const SIZES = [ICON_SIZES.rg, ICON_SIZES.md, ICON_SIZES.lg] as const;
+const SIZES = [ICON_SIZES.sm, ICON_SIZES.rg, ICON_SIZES.lg, ICON_SIZES.xxl] as const;
```

Comment:
Default icon size is `md`. I suggest to either change that, or keep `ICON_SIZED.md` in the array